src/components/titles/index.js

```diff
@@ -75,6 +75,24 @@ function draw(gd, titleClass, options) {
     var fontLineposition = font.lineposition;
     var fontShadow = font.shadow;
 
+    // Get subtitle properties
+    var subtitleProp = options.subtitlePropName;
+    var subtitleEnabled = !!(subtitleProp);
+    var subtitlePlaceholder = options.subtitlePlaceholder;
+
+    var subtitle = cont.title.subtitle;
+    var subtitleTxt = (subtitle && subtitle.text ? subtitle.text : '').trim();
+    var subtitleFont = subtitle && subtitle.font ? subtitle.font : {};
```

Comment:
This is using the same logic for titles.
You may want to add a helper function for both.

Reply:
I considered adding a helper function but decided it would make the code less readable.